README.md

```diff
@@ -1,23 +1,6 @@
-# Streamr cloud development environment
+# Streamr development environment
 
-This repository contains Docker compose files and command line tool `streamr_docker_dev` for setting up a local Streamr cloud development environment. The cloud environment consists of base services (3rd party) and Streamr services (in-house).
-
-![Streamr cloud architecture](high-level.png)
```

Comment:
Might want to remove `high-level.png` and `high-level.xml` files too in this PR.